derp/derp_server.go

```diff
@@ -1398,3 +1413,14 @@ func writePublicKey(bw *bufio.Writer, key *key.Public) error {
 	}
 	return nil
 }
+
+func (s *Server) ServeDebugTraffic(w http.ResponseWriter, r *http.Request) error {
+	s.mu.Lock()
+	defer s.mu.Unlock()
+	if time.Since(s.lastStatTime) < 5*time.Minute {
+		return nil
+	}
+	s.lastStatTime = time.Now()
+	// TODO log stats here
+  return nil
```

Comment:
You should make your editor gofmt-on-save. (Where your gofmt command is actually goimports)

Reply:
hm maybe, I tend to avoid that and I'll always run fmt before any final changes